Data/Models/Academies/LatestOfstedJudgement.cs

```diff
@@ -6,16 +7,56 @@ public class LatestOfstedJudgement
     {
         [DisplayName("School name")]
         public string SchoolName { get; set; }
-        
+
         [DisplayName("Ofsted inspection date")]
         public string InspectionDate { get; set; }
-        
+
         [DisplayName("Overall effectiveness")]
         public string OverallEffectiveness { get; set; }
-        
+
         [DisplayName("Ofsted report")]
         public string OfstedReport { get; set; }
-        
+
+        public string QualityOfEducation { get; set; }
+        public string BehaviourAndAttitudes { get; set; }
+        public string PersonalDevelopment { get; set; }
+        public string EffectivenessOfLeadershipAndManagement { get; set; }
+        public string InspectionEndDate { get; set; }
+
+        public string EarlyYearsProvision { get; set; }
+        public bool EarlyYearsProvisionApplicable { get => OfstedRatingHasData(EarlyYearsProvision); }
+        public string SixthFormProvision { get; set; }
+        public bool SixthFormProvisionApplicable { get => OfstedRatingHasData(SixthFormProvision); }
+
+        public string DateOfLatestInspection
+        {
+            // We have previously been using InspectionDate as the date of the latest full inspection,
+            // however, it's possible for InspectionDate to be null while InspectionEndDate matches
+            // the latest report.
+            get => InspectionDate ?? InspectionEndDate;
+        }
```

Comment:
I found examples where `InspectionDate` was null and `InspectionEndDate` had the correct value, as well as examples where `InspectionEndDate` was null and `InspectionDate` had a value.

I used end date instead of start date because it matches the InspectionDate value